tech/database/cassandra/cassandra_configuration.md

```diff
@@ -0,0 +1,77 @@
+## How to configure Cassandra
```

Comment:
You're missing a header here.

Reply:
Also the page should start with 
```
# Title
```
(not a `## Subtitle`)
Please check our [contribution guide](https://developer-portal.github.io/doc/file-structure.html).